analyzer/src/funTest/assets/projects/external/jgnash-core-expected-output.yml

```diff
@@ -99,6 +99,10 @@ packages:
     path: ""
 - id: "Maven:com.fasterxml:classmate:1.3.0"
   purl: "pkg:maven/com.fasterxml/classmate@1.3.0"
+  authors:
+  - "Brian Langel"
+  - "Tatu Saloranta"
+  - "fasterxml.com"
```

Comment:
This (and similar entries below) stem from `organization -> name` and don't seem useful in the *author* context to me. I think we should experiment with disregarding that field, and looking what else it makes disappear from these expected results. If all disappearing results are in the domain name form, I believe should indeed not look at  `organization -> name`.

Reply:
I just gave this a try. While omitting `organization -> name` gets rid of entries like `"fasterxml.com"`, `"com.mchange"`, and some "duplicates" like `"XStream"` (where we already have `"XStream Committers"`), `"JBoss by Red Hat"` (where we already have `"JBoss.org"`), or `"The Apache Software Foundation"` (where we already have `"Apache Software Foundation"`), it also removes valid entries like `"MetaStuff Ltd."`, `"JUnit"`, or `"Pivotal Software, Inc."`.

So bottom line, I think we need to keep looking at `organization -> name`. However, while looking at these results I'm now wondering whether we should further distinguish between organizations and persons in the `authors` field (similar to [like SPDX does](https://spdx.github.io/spdx-spec/3-package-information/#36-package-originator)), or whether it's fine to mix these. If we mix these here, we later have no chance anymore to separate them, e.g. when *generating* SPDX.

Reply:
I'm also not happy about the "domain name form" entries or duplicates. But if you omit `organization -> name`, useful entries are also omitted. That's why I'm a little overwhelmed by how I should best deal with it. But even if we distinguish between organizations and persons, we will have duplicates or domain-like entries.

Reply:
Yeah, let's just go with the current approach and eventually fine-tune the logic later.